akd/src/storage/tests.rs

```diff
@@ -176,7 +176,7 @@ async fn test_batch_get_items<Ns: Storage>(storage: &Ns) {
                 plaintext_val: Values(value.clone()),
                 version: epoch,
                 label: NodeLabel {
-                    val: 1u64,
+                    val: byte_arr_from_u64(1u64),
```

Comment:
Nit: We can remove the `u64` for consistency with the other calls to `byte_arr_from_u64` in this file :)